core/server/master/src/main/java/alluxio/underfs/UfsStatusCache.java

```diff
@@ -134,6 +134,7 @@ public Collection<UfsStatus> addChildren(AlluxioURI path, Collection<UfsStatus>
     // Update global counters for all InodeSyncStream
     int sizeChange = children.size() - (previousStatuses != null ? previousStatuses.size() : 0);
     DefaultFileSystemMaster.Metrics.UFS_STATUS_CACHE_CHILDREN_SIZE_TOTAL.inc(sizeChange);
```

Comment:
A global counter is bad because the errors will stack up, and after running for some time, these metrics will be totally messed up that the counters don't mean anything at all. The root problem is logged in https://github.com/Alluxio/alluxio/issues/16845